e2e/404.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { test, expect, Page } from '@playwright/test';
+
+test.describe('404 Page', () => {
+  let page: Page;
+
+  test.beforeAll(async ({ browser }) => {
+    page = await browser.newPage();
+    await page.goto('/404');
+  });
+
+  test.afterAll(async () => {
+    await page.close();
+  });
+
+  test('Should have a relevant page title', async () => {
+    await expect(page).toHaveTitle('404 Page Not Found');
```

Comment:
i checked the title of 404 page of the site, i and think it  says `Page not found| freeCodeCamp`, sorry if i checked incorrectly

Reply:
![image](https://github.com/freeCodeCamp/freeCodeCamp/assets/98512995/90a49e3d-2299-4910-b041-ddccd577d222)

This is the 404 page. Should the function contain "Page not found."? I mean like this `await expect(page).toHaveTitle('Page not found.');`